libmamba/include/mamba/specs/version_spec.hpp

```diff
@@ -56,6 +56,13 @@ namespace mamba::specs
          */
         [[nodiscard]] auto has_glob() const -> bool;
 
+        /**
+         * True if the predicate is a simple operator.
+         *
+         * Simple operators are ``=``, ``!=``, ``<``, ``<=``, ``>``, ``>=``.
```

Comment:
Why the double back-ticks? Doxygen handles Markdown so one backtick should be enough?

Reply:
In Sphinx/reStructured which we use in conjunction with Doxygen, it's double back-ticks for code snippets.
Since Markdown support arbitrary number of back-ticks, I often put two to make it universal.